### DISCUSSION TITLE
Run multiple instances of rdio on Windows?

### DISCUSSION THREAD

**Discussion by phillydjdan** [chuot/rdio-scanner#356] [General]
I successfully setup 1 instance of rdio using the default configuration, but i would like to have 2 instances running on 1 Windows computer. I'll admit I know nothing about how to edit any config files. I guess what I need to do is change the url used to access 1 of them. Maybe leave 1 set to 3000 and change the other to 3001? Any help would be appreciated.

**Comment by phillydjdan** [comment]
Disregard, I was able to hex edit the .exe and change the port number.

**Reply by tadscottsmith** [reply]
You should just be able to specify the port number on startup? There's an example in the docs.

https://github.com/chuot/rdio-scanner/blob/master/docs/platforms/windows.md#listening-on-an-alternate-port

**Reply by phillydjdan** [reply]
Those instructions appear to cater to the Linux OS, not Windows, so they are not useful to me.
